internal/Game.py

```diff
@@ -27,16 +27,19 @@ def add_captured_piece(self, captured_piece, position):
         position.add_captured_piece(captured_piece)
 
     def to_string(self):
-        result = "Captured pieces:\n"
-        result += "Black:\n"
+        result = 'Captured pieces:\n'
+        result += 'Black:\n'
         for piece_type in self.captured_pieces[Color.WHITE]:
             count = self.captured_pieces[Color.WHITE][piece_type]
             if count > 0:
                 result += f'{util.piece_type_to_string(piece_type)} x {count}\n'
```

Comment:
In python3, you can use de format() function https://docs.python.org/fr/3.9/library/stdtypes.html#str.format :)